librz/core/cmd_info.c

```diff
@@ -1064,20 +1064,20 @@ RZ_IPI int rz_cmd_info(void *data, const char *input) {
 						if (input[2] == '*') {
 							mode |= RZ_MODE_RIZINCMD;
 						}
-						RBININFO("classes", RZ_CORE_BIN_ACC_CLASSES, NULL);
+						RZBININFO("classes", RZ_CORE_BIN_ACC_CLASSES, NULL);
 						input = " ";
```

Comment:
can you delete this line? ` Value stored to 'input' is never read `

Reply:
Sure.